src/Main.elm

```diff
@@ -129,7 +140,7 @@ update msg model =
                     )
             of
                 Just course ->
-                    ( { model | selectedCourse = Just course }
+                    ( { model | selectedCourse = Just course, selectedCourses = model.selectedCourses ++ [ course ] }
```

Comment:
Maybe `course :: model.selectedCourses` would be better.
(`head :: tail` creates a list by appending head to the front of tail)